install.sh

```diff
@@ -5,3 +5,4 @@ if [ ! -d venv ]; then
 fi
 source venv/bin/activate
 pip install -r requirements.txt
+NEW_RELIC_EXTENSIONS=false pip install --no-binary :all: newrelic==2.82.0.62
```

Comment:
it's also a very old version, required by Python 2?

Reply:
copied from lax. we should definitely look at upgrading it, I'll open a ticket with the affected projects